app/validators/form_password_validator.rb

```diff
@@ -9,13 +11,52 @@ module FormPasswordValidator
               presence: true,
               length: Devise.password_length
 
-    validate :valid_password
+    validate :not_weak_password, if: :password_strength_enabled?
+  end
+
+  ZXCVBN_TESTER = ::Zxcvbn::Tester.new
+
+  private
+
+  def not_weak_password
```

Comment:
I thought about that. I went with what the gem was using, and I was trying to figure out why they made that choice. I'm fine with changing to `strong_password`.